packages/server/lib/ipc/ipc.js

```diff
@@ -1,9 +1,15 @@
-const {
-  contextBridge,
-  ipcRenderer,
-} = require('electron')
+const debug = require('debug')('cypress:server:ipc')
 
-contextBridge.exposeInMainWorld('ipc', {
-  on: (...args) => ipcRenderer.on(...args),
-  send: (...args) => ipcRenderer.send(...args),
-})
+try {
+  const {
+    contextBridge,
+    ipcRenderer,
+  } = require('electron')
+
+  contextBridge.exposeInMainWorld('ipc', {
+    on: (...args) => ipcRenderer.on(...args),
+    send: (...args) => ipcRenderer.send(...args),
+  })
+} catch {
+  debug('running as a node process, not an electron process')
```

Comment:
Instead of doing a `try..catch` in multiple places and calling `debug` with the same or similar strings each time, could this be refactored to use a helper function to share the logic and debugging statement and make it more clear what the code is doing?

Some helper function:
```js
const isRunningAsElectronProcess = ({ debug } = {}) => {
  const isElectronProcess = /* some code goes here */
  if (!isElectronProcess && debug) {
    debug('running as a node process without xvfp due to ELECTRON_RUN_AS_NODE env var')
  }
  return isElectronProcess
}
```

Other Cypress code:
```js
const debug = require('debug')('cypress:server:ipc')

if (isRunningAsElectronProcess({ debug })) {
  const {
    contextBridge,
    ipcRenderer,
  } = require('electron')

  contextBridge.exposeInMainWorld('ipc', {
    on: (...args) => ipcRenderer.on(...args),
    send: (...args) => ipcRenderer.send(...args),
  })
}

```

Reply:
Done.